book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] We want to be able to have full access to the rentals, not just read it. Kindly add `:rentals` to the `attr_accessor` line.